pkgs/development/python-modules/nbqa/default.nix

```diff
@@ -0,0 +1,64 @@
+{ autopep8
+, black
+, buildPythonPackage
+, fetchPypi
+, flake8
+, ipython
+, isort
+, jupytext
+, lib
+, mypy
+, pylint
+, pythonAtLeast
+, pythonOlder
+, pyupgrade
+, ruff
+, tokenize-rt
+, tomli
+}:
+buildPythonPackage rec {
```

Comment:
I think this is an application, not a library.
How about using buildPythonApplication if we don't intend to import?

Reply:
It is meant to be used as both a library and an application.

Reply:
OK, then would you please prefix your commit message and PR title with `python310Packages`?

Reply:
Oh shoot, I was looking at the wrong file & branch! Please disregard my comments; I'll apply your suggestions or post new replies here.

Reply:
Applied your suggestion, thank you!